### PR TITLE
Huge refactor before release

### PR DESCRIPTION
**Pull request by AlexandreSajus**
Restructured the working tree
Changed OCB to Pyflow
Removed useless code and files
Added a window title

### CODE REVIEW

**Comment by AlexandreSajus** [comment]
I fucked up the badges, gimme a second

**Comment by AlexandreSajus** [comment]
Codacy can't be accessed by clicking on the badge on the README, it returns: "We couldn't find gh/MathisFederico/opencodeblocks", can't find why

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,26 +43,26 @@ The comments and docstrings should preferably follow [these](https://google.gith
...
+ | :memo: `:memo:`                                           | When you wrote documentation.                      |
+ | :umbrella: `:umbrella:`                                   | When you improved coverage                         |
+ | :twisted_rightwards_arrows: `:twisted_rightwards_arrows:` | When you merge a branch                            |
 
 This section was inspired by [This repository](https://github.com/schneegans/dynamic-badges-action).
 
 ## Creating a new block type
 
-You can checkout [this commit](https://github.com/MathisFederico/OpenCodeBlocks/commits/2305e3c92d88d2fd32644e7eab4c4e40246675d3) which contains the minimal amount of code required to
+You can checkout [this commit](https://github.com/MathisFederico/Pyflow/commits/2305e3c92d88d2fd32644e7eab4c4e40246675d3) which contains the minimal amount of code required to
```
Comment (MathisFederico) [inline CONTRIBUTING.md:65]:
```suggestion
You can checkout [this commit](https://github.com/Bycelium/Pyflow/commits/2305e3c92d88d2fd32644e7eab4c4e40246675d3) which contains the minimal amount of code required to
```

File: blocks/empty.pfb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "title": "Empty",
+    "title": "Code",
```
Comment (MathisFederico) [inline blocks/empty.pfb:2]: Why this change ?

File: pyflow/blocks/widgets/__init__.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Pyflow an open-source tool for modular visual programing in python
+# Copyright (C) 2021 Mathïs FEDERICO <https://www.gnu.org/licenses/>
+
+""" Module for the OCB Blocks Widgets. """
+
+from pyflow.blocks.widgets.blocksplitter import OCBSplitter
```
Comment (MathisFederico) [inline pyflow/blocks/widgets/__init__.py:6]: Aren't those removed ?

**Review by MathisFederico — COMMENTED** [review]
A few questions:

File: blocks/empty.pfb (hunk 2 above)
Comment (AlexandreSajus) [reply blocks/empty.pfb:2]: For a user, when right-clicking on the interface they had the choice between "Empty Block" and "Markdown Block", I replaced Empty with Code for clarity purposes

File: pyflow/blocks/widgets/__init__.py (hunk 3 above)
Comment (AlexandreSajus) [reply pyflow/blocks/widgets/__init__.py:6]: I am not sure, what do you want to do?

Comment (MathisFederico) [reply pyflow/blocks/widgets/__init__.py:6]: No sorry I've mistaken this for Slider xD

**Review by MathisFederico — APPROVED** [review]
